apps/prairielearn/src/schemas/infoAssessment.ts

```diff
@@ -278,15 +308,17 @@ export const AssessmentJsonSchema = z
       .default(false),
     shuffleQuestions: z
       .boolean()
-      .describe('Whether the questions will be shuffled in the student view of an assessment')
-      .optional()
-      .default(false),
+      .describe(
```

Comment:
This wasn't supposed to have a default. There is logic based on if this wasn't set.